datestuffs.js

```diff
@@ -0,0 +1,109 @@
+//Sam Radakovitz
+//Module 2
+
+/* exported dateStuff */
+var dateStuff = (function start() {
+
+    //private date var
+  var myDate;
+
+    //public: store date
+  function setDate(theDate) {
+
+        //check to see if i got something
+    if (theDate) {
+
+            //date passed in, use it
+      myDate = new Date(theDate);
```

Comment:
Better would be to validate the input for this method before trying to instantiate a date. The user could pass in a value that causes an invalid date to be instantiated by your code.